src/admin/model/payment/omise.php

```diff
@@ -201,4 +210,8 @@ private function _retrieveOmiseKeys() {
 
         return $keys;
     }
+    
+    public function get_default_auto_capture() {
```

Comment:
@nimid a method name should be `getDefaultAutoCapture` instead. 
ref. https://github.com/opencart/opencart/wiki/Coding-standards#correct-5